src/Flash/Flash.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Sitchco\Flash;
+
+use Sitchco\Framework\Core\Module;
+use Sitchco\Utils\Hooks;
+
+/**
+ * class Flash
+ * @package Sitchco\FlashMessage
+ */
+class Flash extends Module
+{
+    protected AdminNotificationService $service;
+    public function __construct(AdminNotificationService $service)
+    {
+        $this->service = $service;
+    }
+
+    public function init(): void
+    {
+        Hooks::callOrAddAction('admin_notices', [$this, 'render']);
+        Hooks::callOrAddAction('shutdown', [$this, 'shutdown']);
+    }
+
+    /**
+     * Renders and outputs stored admin notifications.
+     *
+     * @return array The list of notifications displayed.
+     */
+    public function render(): array
+    {
+        $allNotifications = array_unique($this->retrieveStoredNotifications());
+        error_log(var_export($allNotifications, true));
+        foreach ($allNotifications as $notification) {
+            echo $notification . "\n";
+        }
+
+        // Clear notifications after rendering
+        $this->service->clearNotifications();
```

Comment:
This isn't really needed, since the service is just for dispatching and has no bearing on rendering on the next page load.